docs/standard/generics.md

```diff
@@ -9,15 +9,15 @@ ms.assetid: a315b111-8e48-446c-ab19-acb6405894a7
 ---
 # Generic Types (Generics) Overview
 
-We use generics all the time in C#, whether implicitly or explicitly. When you use LINQ in C#, did you ever notice that you are working with IEnumerable<T>? Or if you ever saw an online sample of a "generic repository" for talking to databases using Entity Framework, did you see that most methods return IQueryable<T>? You may have wondered what the **T** is in these examples and why is it in there?
+We use generics all the time in C#, whether implicitly or explicitly. When you use LINQ in C#, did you ever notice that you're working with IEnumerable<T>? Or if you ever saw an online sample of a "generic repository" for talking to databases using Entity Framework, did you see that most methods return IQueryable<T>? You may have wondered what the **T** is in these examples and why is it in there?
```

Comment:
Unrelated to your PR:
- This is a .NET Standard topic, but it's focus is on C#. Yet all of its examples are language-independent. It would be good to remove references to C#.
- The last sentence should end in a period, not a question mark.